tests/chainer_tests/functions_tests/noise_tests/test_simplified_dropconnect.py

```diff
@@ -98,4 +98,55 @@ def test_backward_gpu_nobias(self):
                             None, cuda.to_gpu(self.gy))
 
 
+class TestSimplifiedDropconnectNotBatchwiseMask(unittest.TestCase):
+
+    def setUp(self):
+        self.W = numpy.random.uniform(
+            -1, 1, (2, 3)).astype(numpy.float32)
+        self.b = numpy.random.uniform(
+            -1, 1, 2).astype(numpy.float32)
+
+        self.x = numpy.random.uniform(-1, 1, (4, 3)).astype(numpy.float32)
+        self.gy = numpy.random.uniform(-1, 1, (4, 2)).astype(numpy.float32)
+        self.y = self.x.dot(self.W.T) + self.b
+        self.check_forward_options = {}
+        self.check_backward_options = {}
+
+    def check_forward(self, x_data, W_data, b_data):
+        # Check only data type, y is tested by SimplifiedDropconnect link test.
+        x = chainer.Variable(x_data)
+        W = chainer.Variable(W_data)
+        b = chainer.Variable(b_data)
+        func = simplified_dropconnect.SimplifiedDropconnect(0.5, False)
+        y = func(x, W, b)
+        self.assertEqual(y.data.dtype, numpy.float32)
+        self.assertEqual(func.mask.shape, W.shape)
```

Comment:
Could you add this shape-check to the test case that uses sample-wise masks (i.e. `TestSimplifiedDropconnect`)?